test/faker/datetime_test.exs

```diff
@@ -5,19 +5,21 @@ if Version.match?(System.version(), ">= 1.3.0") do
     @microseconds_per_day 86400000000
 
     test "forward/1" do
+      now = now()
```

Comment:
I do understand why you want a variable named `now` since it's nice and short, but the function is also already called `now`. I'd suggest just using `now = DateTime.utc_now()` and remove the `now` function.